nixui/graphics/navlist.py

```diff
@@ -69,6 +69,10 @@ def __init__(self, option, use_fancy_name=True, use_child_count=True, extra_text
         self.option_tree = api.get_option_tree()
 
         child_count = len(self.option_tree.children(self.option)) if use_child_count else None
+
+        self.setText(str(self.option) + ("." if child_count > 0 else ""))
+
+        """
```

Comment:
Please remove this commented portion since it's being replaced.

Reply:
please help yourself ; ) sorry, i dont have time to "fix all the code".
feel free to use my patch as a start